lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,20 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = CreditRequirement.get_course_requirement(
```

Comment:
CreditRequirement.get_course_requirement() raises DoesNotExist exception. Please add the try catch block around this code so that it should not break the code flow if something bad happens